src/main/g8/default.properties

```diff
@@ -0,0 +1,5 @@
+name =
```

Comment:
It can't be renamed, it's a reserved word in the giter8 template which determines what the folder is called. 

We can definitely add a full_name variable in. I've set it to default to test-repo -> Test Repo but that can then be overridden.

Reply:
Do we want our lambda repos to end `-lambda` so it's clear that the repo is for a lambda?

Reply:
I don't know. It's possible that the code could be shifted to run somewhere other than lambda in the future and then that's just confusing. It shouldn't really matter reading the code where it runs. What do you think @techncl ?

Reply:
Also, because the reserved `name` is used for the folder name, I don't think you can add a suffix to it like that.

Reply:
It would make it easier when searching through Github but then again, in point out that it's a Lambda, it could/should be made clear in the "description" (which is also searchable). Maybe instead of an empty string as the default for the "description", we could add "Lambda" or "A Lambda" to encourage people to complete the description?

Reply:
That makes sense, I've added in a default description of "A Lambda"